packages/core/src/actors.ts

```diff
@@ -281,6 +281,66 @@ export function fromObservable<T, TEvent extends EventObject>(
   return behavior;
 }
 
+/**
+ * Creates an event observable behavior that listens to an observable
+ * that delivers event objects.
+ *
+ *
+ * @param lazyObservable A function that creates an observable
+ * @returns An event observable behavior
+ */
+export function fromEventObservable<T extends EventObject>(
```

Comment:
bikesheddable alternative name: `fromObservableEvents`

I'm OK with the current name, was just thinking if this could be named differently or not

Reply:
At first, I did call it `fromObservableEvents` but it made it awkward to answer the question "What does this behavior create?"

It's more succinct to say "This behavior creates an event observable" than "This behavior creates an observable that delivers events"